epub34/authoring/index.html

```diff
@@ -6215,6 +6215,22 @@ <h4>Introduction</h4>
 									<p>Some types of screens will render animations and transitions poorly (e.g., those
 										with high latency).</p>
 								</li>
+								<li>
+									<p>
+										Creators should be aware that a reading systems usually override some aspects of 
+										the creator's style settings; creators may want to consult the reading systems' 
+										user agent style sheets, if publicly available. Furthermore, reading systems may
+										also modify some rendering aspects as a result of user interaction (e.g., choice 
+										of fonts, text justification, or foreground and background colors).
+									</p>
+								</li>
+								<li>
+									<p>
+										Creators should avoid using [[html]] [^html-global/style^] attributes in content documents. 
```

Comment:
This really sounds like normative guidance. In any case, similar to my previous comment it's out of place in a note about poor support for CSS features.

It's also probably more of a consideration for the accessibility specification or techniques, although it would really help to know more about how reading systems modify the display. Inline styles (particularly author !important ones) can be problematic, but it's historically been more of an issue with applying user style sheets in browsers.

Reply:
This is something we would fail a GCA certification if we see inline style=... within an EPUB as then this style cannot be overridden. Letting publishers know this is not the correct way to add styling to an EPUB is very important.

Reply:
> within an EPUB as then this style cannot be overridden

Unless the reading system is changing styles by modifying the dom, or the epub is preprocessed to remove inline styles, or...? That's why it would help to know how reading system styling is applied.

What is the process for changing the display when a user picks custom display properties?

Inline styling, especially with !important, is bad on more levels than just accessibility, so I'm not disagreeing with you in principle. I'm just not sure this is an authoring spec-level normative restriction, which is what it currently sounds like but masked in a note.